test/diffTest.js

```diff
@@ -200,6 +200,36 @@ describe('#diffLines', function() {
   });
 });
 
+// Line Diff
+describe('#TrimmedLineDiff', function() {
+  it('should diff lines', function() {
+    var diffResult = diff.diffTrimmedLines(
+      'line\nold value\nline',
+      'line\nnew value\nline');
+    diff.convertChangesToXML(diffResult).should.equal('line\n<ins>new value\n</ins><del>old value\n</del>line');
+  });
+  it('should the same lines in diff', function() {
+    var diffResult = diff.diffTrimmedLines(
+      'line\nvalue\nline',
+      'line\nvalue\nline');
+    diff.convertChangesToXML(diffResult).should.equal('line\nvalue\nline');
+  });
+
+  it('should ignore shorespace', function() {
```

Comment:
On line 188, in the tests for #diffLines:

<code>it('should handle shorespace', function() {</code>

I was reversing that test. Shorespace refers to the leading and trailing whitespace. I think both tests should be titled similarly, so they can be easily juxtaposed. Should I switch both to say "leading and trailing whitespace" instead of "shorespace"?